sams/sampler/FSStats.py

```diff
@@ -97,7 +97,10 @@ def do_sample(self):
     def sample(self):
         logger.debug("sample()")
         if self.fsstat:
-            self.store(self.fsstat.sample())
+            fsstat_sample = self.fsstat.sample()
+            self._most_recent_sample = self._sample_wrapping(fsstat_sample)
+            self.store(fsstat_sample)
+
```

Comment:
```suggestion
```

Not sure why an extra line was added here, not supposed to be there. Not fixing it for now to keep the commit clean.